pageserver/src/layered_repository/timeline.rs

```diff
@@ -269,11 +268,18 @@ pub struct LayeredTimeline {
     // though lets keep them both for better error visibility.
     pub initdb_lsn: Lsn,
 
-    // Initial logical size of timeline (if known).
-    // Logical size can be copied from ancestor timeline when new branch is create at last LSN
-    pub init_logical_size: Option<usize>,
+    /// When did we last calculate the partitioning?
```

Comment:
I don't get this description. Seeing "when", I expect the variable to be timestamped. Should this be something that mentions "the last partitioning at some LSN"?

Reply:
I see your point. I think of LSN as the time dimension here, but you're right, it's not self-evident. This PR just moves the code as is, so I'll try to clarify that as a separate PR.